scrapers/utils/events.py

```diff
@@ -1,3 +1,21 @@
+import hashlib
+import uuid
+
+
+def hash_event_key(event_key_str):
```

Comment:
I could see this being useful across more than just events. Maybe this belongs in the __init__.py file with a more generic name?

Reply:
Made generic in `__init__` file in [72ea4a3](https://github.com/openstates/openstates-scrapers/pull/4643/commits/72ea4a3ae7d929ca1af9fcf532074299f48fb438)